EDI update

parkyijoo opened the pull request:
EDI script update

aostropolets commented:
@parkyijoo Thank you very much for the contribution and for your time!
Could you please add:
- insert of mappings to concept_relationship_stage/concept_stage_manual (wherever you stored the mappings); if insert was done from another manual table it is fine just to write one insert from that table (for the purpose of denoting that mappings were added)
- your name (and others who should be added) to the list of the authors of the script (we want to explicitly celebrate each and everyone who contributes :))

parkyijoo commented:
@aostropolets Hi, Anna! I have just pushed a new update. Overall, I simplified unnecessary procedures in the readme and modified the content to include the points you and @TinyRickC137 mentioned. Thank you! :)
